lib/engine/game/g_1877_stockholm_tramways/game.rb

```diff
@@ -139,10 +143,58 @@ class Game < Game::Base
            'sl_trigger' => ['SL Trigger', 'SL will form at end of OR, game ends at end of following OR set'],
          ).freeze
 
+        def setup
+          @sl = nil
```

Comment:
What is s1?

Reply:
SL is the name of the end of game event, like LNER in 1862. I have however not implemented it's logic yet.